packages/xs-extra/wsproxy.master/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "wsproxy"
+version: "1.1.0"
```

Comment:
I am testing it here: https://travis-ci.org/mseri/xs-opam/builds/211281238
We need to fix lots of opam files, I guess it's better to do that in another PR